### PR TITLE
Modifed unittest testscript

### PR DESCRIPTION
**Pull request by fiona-gladwin**

### CODE REVIEW

File: utilities/rocAL/rocAL_unittests/pixel_comparison/image_comparison.py [hunk 1]
```diff
@@ -4,98 +4,144 @@
...
-            img2 = Image.open(rocal_file_path)
+    # Open the two images
+    ref_output_path = sys.argv[1]
+    rocal_output_path = sys.argv[2]
 
-            # Check if the images have the same dimensions
-            if img1.size != img2.size:
-                logging.info("The images have different sizes.")
-                exit()
+    if not (os.path.exists(ref_output_path) and os.path.exists(rocal_output_path)):
```
Comment (shobana-mcw) [inline utilities/rocAL/rocAL_unittests/pixel_comparison/image_comparison.py:57]: This should be an or condition ryt?

File: utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp [hunk 2]
```diff
@@ -747,7 +750,7 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
     {
         std::cout << ">>>>>>> Running "
                   << "rocalResizeCropMirrorFixed" << std::endl;
-        image1 = rocalResizeCropMirrorFixed(handle, image0, 300, 300, true, 250, 250, 0);
+        image1 = rocalResizeCropMirrorFixed(handle, image0, 300, 300, true, 250, 250, mirror);
```
Comment (shobana-mcw) [inline utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp:753]: Why for fixed we are getting rocalintparam? We should be getting unit ryt?

Comment (fiona-gladwin) [reply utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp:753]: The 2 API rocalCropMirrorNormalize and rocalResizeCropMirrorFixed, both are accepting mirrors as RocalIntParam only

File: utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp [hunk 3]
```diff
@@ -360,9 +360,12 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
 
     RocalImage image0 = input1;
     // RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false); // uncomment when processing images of different size
-
     RocalImage image1;
-
+    
+    if((test_case == 48 || test_case == 98) && rgb == 0) {
```
[the remark this answers is not in the document]
Comment (fiona-gladwin) [reply utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp]: @Akilesh2 please add ColorTwist here and remove the condition in python code

**Pull request merged**